pygments_tiger.py

```diff
@@ -24,9 +24,10 @@ class TigerLexer(RegexLexer):
             (r'[\n\r\s]+', Text),
         ],
         'comment': [
+            (r'[^*/]', Comment),
             (r'/\*', Comment, '#push'),
             (r'\*/', Comment, '#pop'),
-            (r'.+', Comment),
+            (r'[*/]', Comment),
```

Comment:
I am reading the document, and see that there is a `Comment.Multiline` token type. Would it be pertinent to use it instead of `Comment`?

Reply:
Since there is only one way to comment code in Tiger it does not change anything if we specify the `Multiline` or not. I changed it to make it clearer.